zk/src/archive/web/js/zk/widget.js

```diff
@@ -2161,6 +2162,10 @@ wgt.$f().main.setTitle("foo");
 	 * @return zk.Widget this widget.
 	 */
 	setZIndex: _zkf = function (zIndex, opts) {
+		if (!opts.floatZIndex)

```

Comment:
Add this opt option to jsdoc

Reply:
add,
	/** Sets the Z index.
	 * @param int zIndex the Z index to assign to
	 * @param Map opts if opts.fire is specified, the onZIndex event will be triggered. If opts.floatZIndex is false, 
	 * represent it is not from setFloatZIndex, so the userZIndex mey be true
	 * @return zk.Widget this widget.
	 */